inlong-sort/sort-core/src/main/java/org/apache/inlong/sort/parser/impl/NativeFlinkSqlParser.java

```diff
@@ -82,7 +82,7 @@ public ParseResult parse() {
         Preconditions.checkNotNull(tableEnv, "tableEnv is null");
         List<String> createTableSqls = new ArrayList<>();
         List<String> insertSqls = new ArrayList<>();
-        String[] statementSet = statements.split(";");
+        String[] statementSet = statements.split(";(\\r?\\n|\\r)");
```

Comment:
Maybe it is better for ';(\\r?\\n|\\r|\\n)'?

Reply:
In most case, `;(\\r?\\n|\\r)`  is ok. But If there is a sql statement like this `concat(';\n','b')`, may be problem.